item.gs

```diff
@@ -6,14 +6,15 @@
  * @constructor
  */
 function Item(i, header, row, column, sheet) {
-  this.i = i;
+  this.fields = {};
   this.header = header;
```

Comment:
Why not make the same with this.header? (this.table.header)

Reply:
Great idea! Even though won't be retrocompatible... I don't think it's a big issue